src/guiapi/src/window_frame.c

```diff
@@ -36,18 +37,28 @@ void window_frame_event(window_frame_t *window, uint8_t event, void *param) {
     case WINDOW_EVENT_ENC_DN:
         dif = (int)param;
         id = window_focused();
-        while (dif--)
+        while (dif--) {
```

Comment:
i think those brackets  `{` after `while (dif--)` are wrong
seems like 
```
if (id >= 0)
            window_set_focus(id);
```
was intended to run only once

Reply:
my bad - sorry